src/Vimeo/Vimeo.php

```diff
@@ -20,48 +20,49 @@
  *   limitations under the License.
  */
 
-if (!function_exists('json_decode')) {
+if (!function_exists('json_decode'))
+{
     throw new \Exception('We could not find json_decode. json_decode is found in php 5.2 and up, but may be missing on some Linux systems due to licensing conflicts. If you are running ubuntu try "sudo apt-get install php5-json".');
 }
 
 class Vimeo
 {
-    const ROOT_ENDPOINT = 'https://api.vimeo.com';
-    const AUTH_ENDPOINT = 'https://api.vimeo.com/oauth/authorize';
-    const ACCESS_TOKEN_ENDPOINT = '/oauth/access_token';
-    const CLIENT_CREDENTIALS_TOKEN_ENDPOINT = '/oauth/authorize/client';
-    const REPLACE_ENDPOINT = '/files';
-    const VERSION_STRING = 'application/vnd.vimeo.*+json; version=3.2';
-    const USER_AGENT = 'vimeo.php 1.2.6; (http://developer.vimeo.com/api/docs)';
-    const CERTIFICATE_PATH = '/certificates/vimeo-api.pem';
-
-    private $_client_id = null;
-    private $_client_secret = null;
-    private $_access_token = null;
-
-    protected $_curl_opts = array();
-    protected $CURL_DEFAULTS = array();
+    const rootEndPoint = 'https://api.vimeo.com';
```

Comment:
Adding StyleCI is definitely something we'd like to do after getting the code to a consistent standard.